### PR TITLE
Code dir name from repo name + enable virtualenv pytest

### PR DESCRIPTION
**Pull request by jackel119**

### CODE REVIEW

File: hypothesisfuzzer/fuzz_server.py [hunk 1]
```diff
@@ -24,7 +24,9 @@ def __init__(self, config_path='config.yml'):
         self._load_config(config_path)
         self.db = SQLAlchemy(self.app)
         self.current_fuzzing_task = None
-        self.fuzzer = RepoFuzzer(self.config)
+
+        for name, repo_config in self.config['repos']:
+            self.fuzzer = RepoFuzzer(name, repo_config)
```
Comment (zicongma) [inline hypothesisfuzzer/fuzz_server.py]: Many fuzzers?  Don't quite get it plz explain.

Comment (romdespoel) [reply hypothesisfuzzer/fuzz_server.py]: We want to give users the option to fuzz multiple repos in parallel. This will be an option in the config file.

Comment (jackel119) [reply hypothesisfuzzer/fuzz_server.py]: This is to support a future ticket where we allow multiple fuzzers. The second line will probably be changed to being added to a dictionary.

File: hypothesisfuzzer/fuzzing/repo_fuzzer.py [hunk 2]
```diff
@@ -57,17 +59,20 @@ def get_errors():
...
+
+        os.makedirs(self.name)
+        GitRepo.clone_from(git_url, self.name)
+
+    def _create_venv(self):
 
-        os.makedirs(folder_name)
-        GitRepo.clone_from(git_url, folder_name)
+        os.chdir(self.name)
+        assert os.path.basename(os.getcwd()) == self.name
```
Comment (romdespoel) [inline hypothesisfuzzer/fuzzing/repo_fuzzer.py]: Why do we need this assertion?

Comment (jackel119) [reply hypothesisfuzzer/fuzzing/repo_fuzzer.py]: So we don't get anymore bugs with being in the wrong OS (if we do, we will know immediately).

**Pull request merged**